Gemfile

```diff
@@ -48,7 +48,8 @@ gem 'mini_magick'
 gem 'mysql2'
 
 # Tables
-gem 'wice_grid'
+gem 'wice_grid', '3.6.0.pre4'
```

Comment:
I wish the developer moved this version over to 3.6.1 or something. This specific version is needed because the master branch has an older way of using associations in the table, specifically related to filters and joined tables. See https://github.com/leikind/wice_grid/wiki/What's-New-In-3.6.0#new-api-for-joined-tables.

I started using the older API and master branch, but ran into some problems with items from associations and not being able to use those contents for filtering. So instead I replaced the gem with the newest version recommended by the developer and it filtering works with items from a joined table.